src/extension.ts

```diff
@@ -56,6 +61,8 @@ export async function activate(context: vscode.ExtensionContext) {
 
     ext.context = context
     await activateLogger(context)
+    const toolkitEnvDetails = getToolkitEnvironmentDetails()
```

Comment:
Move this into the try block to reduce the risk of a bad extension activation

Reply:
Thanks, I moved the call into the try block.